lib/robots/dor_repo/gis_delivery/load_raster.rb

```diff
@@ -45,7 +45,7 @@ def normalizer
         end
 
         def rootdir
-          @rootdir ||= GisRobotSuite.locate_druid_path bare_druid, type: :workspace
+          @rootdir ||= GisRobotSuite.locate_druid_path bare_druid, type: :stage
```

Comment:
I'm confused, why did we need to change this logic?

Reply:
@edsu because `:workspace` is where the files are placed for the dor mount (i.e. the end up in `/dor/workspace/{druid tree}...` for accessioning). Before this PR that space was also being used (for some reason) to create the temp normalized files, which doesn't seem right to me, created temp file in the dor space... I also believe I ran into an error here that the files weren't found (but I could be mistaken)...

My preference would be to go with this as is, since creating those files on the dor mount seems incorrect to me, and do a follow up if we decide there is/was a reason for it.